src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final Random RANDOM = new Random();
+    private static final ColorSupplier COLOR_SUPPLIER = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        Type figures = Type.values()[new Random().nextInt(Type.values().length)];

```

Comment:
As @d1sam mentioned, generally here you already have `RANDOM` class-level variable, so why not to use, instead of
`Type.values()[new Random().nextInt(Type.values().length)];`
Also would say `figures` should be changed to `figure`, as it is single one here.